lib/matplotlib/tests/test_colorbar.py

```diff
@@ -552,10 +552,10 @@ def test_colorbar_lognorm_extension(extend):
 
 def test_colorbar_powernorm_extension():
     # Test that colorbar with powernorm is extended correctly
-    f, ax = plt.subplots()
-    cb = Colorbar(ax, norm=PowerNorm(gamma=0.5, vmin=0.0, vmax=1.0),
-                  orientation='vertical', extend='both')
-    assert cb._values[0] >= 0.0
```

Comment:
This check came in with https://github.com/matplotlib/matplotlib/pull/11610, but from that PR it seems like it's enough just to add the colorbar and check no error or warning is raised. It's expected that `_values` now has a value < 0 with this PR, so I don't think it's meaningful to do this check any more.